webapp/src/selectors/index.ts

```diff
@@ -39,26 +42,26 @@ function mapPrsToDetails(prs, details) {
     });
 }
 
-export const getPluginState = (state) => state[`plugins-${manifest.id}`];
+export const getPluginState = (state: GlobalState) => state[`plugins-${manifest.id}` as pluginStateKey];
 
 export const getSidebarData = createSelector(
     getPluginState,
-    (pluginState) => {
+    (pluginState: PluginState) => {
```

Comment:
We can do this to avoid using `as` below

```suggestion
    (pluginState: PluginState): SideBarData => {
```